apps/prairielearn/src/pages/enroll/enroll.html.ts

```diff
@@ -75,6 +75,12 @@ export function Enroll({
             </tbody>
           </table>
         </div>
+        <div class="card-footer">
+          <p>
+            Course not showing up? Courses created after November 2025 that are using the modern
+            publishing system are not shown here, and you should enroll via the home page.
```

Comment:
Realistically this will mean nothing to students and could just confuse them, I'm inclined to remove this.